app/src/main/res/values/dimens.xml

```diff
@@ -23,5 +23,11 @@
     <dimen name="dp_30">30dp</dimen>
     <dimen name="dp_18">18dp</dimen>
     <dimen name="dp_200">200dp</dimen>
+    <dimen name="dp_32">32dp</dimen>
+    <dimen name="dp_36">36dp</dimen>
+    <dimen name="sp_16">16sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_16` appears to be unused
 Unused resources make applications larger and slow down builds.